### ISSUE TITLE
Unable to run applications with anti-cheat systems such as Valorant

### ISSUE THREAD

**Issue by grillecon** [dartraiden/NVIDIA-patcher#69]
The only games I play that have built-in anti-cheat are Valorant. I've attempted to install the new driver 546.17, but it still only displays a black screen. Is there any solution to fix this issue? I believe this problem may be due to the anti-cheat blocking the graphics card from running in this game, as I can play Genshin Impact and Forza Horizon 5 without any issues.

**Comment by Achtungboi** [comment]
Same problem. If i solve it, i will write you.

**Comment by Achtungboi** [comment]
https://github.com/dartraiden/NVIDIA-patcher/issues/95#issuecomment-1979610809

**Comment by Achtungboi** [comment]
Not best solution, but valorant works good

**Comment by grillecon** [comment]
Thanks for the solution you gave, but recently I found out that driver 531.41 was the lastest driver that runs perfectly fine with Valorant. You can try that also.

**Comment by Achtungboi** [comment]
I have a p104-100 with gt710, so the latest driver for me is 472.12, but it doesn't work with valorant(

**Comment by dartraiden** [comment]
Unfortunately, it is impossible to maintain perfect compatibility with anti-cheats, because some of them want the original Nvidia digital signature.

**Comment by dartraiden** [comment]
I can try signing the files with a different certificate if you're willing to test it.

**Comment by grillecon** [comment]
> I can try signing the files with a different certificate if you're willing to test it.

Of course, I'd love to! If there's still a glimmer of hope, I want to give it a try!

**Comment by Achtungboi** [comment]
Мне тоже будет не сложно их протестить

**Comment by dartraiden** [comment]
Compatibility with Valorant should be fixed in 572.75

**Comment by grillecon** [comment]
> Compatibility with Valorant should be fixed in 572.75

Thanks for solving this problem! Unfortunately, I had already sold my P106-100. Now I’m using an RX 580 8GB 2048SP that I bought from a miner at a very cheap price. I just had enough of the P106-100—I need a real graphics card! 😂

**Comment by dartraiden** [comment]
Just tested: Valorant is playable on P106-100 + HD630
